File: SDKs/Aspose.Tasks_Cloud_SDK_for_Java/src/main/java/com/aspose/tasks/model/OutlineCodeResponse.java
package com.aspose.tasks.model;

import com.aspose.tasks.model.OutlineCodeDefinition;
public class OutlineCodeResponse {
  private OutlineCodeDefinition OutlineCode = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getOutlineCode
	 * Gets OutlineCodeDefinition
	 * @return OutlineCode
	 */
  public OutlineCodeDefinition getOutlineCode() {
    return OutlineCode;
  }

	/**
	 * setOutlineCode
	 * Sets OutlineCodeDefinition
	 * @param OutlineCode OutlineCodeDefinition
	 */
  public void setOutlineCode(OutlineCodeDefinition OutlineCode) {
    this.OutlineCode = OutlineCode;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class OutlineCodeResponse {\n");
    sb.append("  OutlineCode: ").append(OutlineCode).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Tasks_Cloud_SDK_for_.NET/src/Com/Aspose/Tasks/Model/OutlineCodeResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Tasks.Model {
  public class OutlineCodeResponse {
    public OutlineCodeDefinition OutlineCode { get; set; }

    public string Code { get; set; }

    public string Status { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class OutlineCodeResponse {\n");
      sb.Append("  OutlineCode: ").Append(OutlineCode).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
